src/core/services/slack.ts

```diff
@@ -94,10 +95,10 @@ export async function fetchSlackUserFromGitlabUser({
 export async function fetchSlackUserFromGitlabUsername(
   username: string
 ): Promise<SlackUser | undefined> {
-  return fetchSlackUserFromEmails([
-    `${username}@manomano.com`,
-    `${username}@prt.manomano.com`,
-  ]);
+  const emails = EMAIL_PATTERNS.split(',').map(
+    (emailPattern) => `${username}${emailPattern}`
```

Comment:
Hang on… does this mean that this method only works if your Gitlab username and the part of your email address that is before the `@` match exactly?

Reply:
I did not change anything to the actual behaviour. I'm passing an array of string as before

Reply:
Yes you are right!

Reply:
I will open an issue for this need

Reply:
I open this one #13